.github/workflows/pull_request.yml

```diff
@@ -122,6 +122,19 @@ jobs:
         with:
           python-version: ${{ matrix.python-version }}
 
+      # Custom tag extractor to find Brewtils Branch
```

Comment:
The code coverage action needs this in cases where a specific brewtils version is required (this same thing happens on the regular unit test action).  Since this PR is one such case, I just added it here.